test/integration/py/Pipfile

```diff
@@ -1,6 +1,6 @@
 [[source]]
 name = "arti-pip-mirror"
-url = "https://arti.dev.cray.com:443/artifactory/api/pypi/pypi-remote/simple/"
+url = "https://arti.hpc.amslabs.hpecorp.net:443/artifactory/api/pypi/pypi-remote/simple/"
```

Comment:
Is this PIP mirror actually being used when integration tests are running? 

If it was I would expect to see the integration tests to fail as the Github running our tests run on should not have access to the internal artifactory instance. Shoudn't we be using an pypi mirror hosted on the Agol60 artifactory or just go up stream?

Looks like the fw-loader Pipfile doesn't have any sources defined (which I think means it pulls it from PyPi directly).: https://github.com/Cray-HPE/hms-firmware-action/blob/master/cmd/fw-loader/Pipfile

Reply:
Mikhail set up a pypi mirror for us in algol60 so I've updated to that location. It seems to work for the packages that are pulled in here for the integration tests.